### ISSUE TITLE
What if I told you we could support both relative/fixed formats?

### ISSUE THREAD

**Issue by ekinhbayar** [ekinhbayar/IntervalParser#17]
Just an idea but... what do you think about making this library support all kinds of formats? Currently it is mainly for intervals but wouldn't it be great if this is literally a TimeParser (or DateTimeParser)?
See [Compound Formats](http://php.net/manual/en/datetime.formats.compound.php) and [Relative Formats](http://php.net/manual/en/datetime.formats.relative.php)

**Comment by PeeHaa** [comment]
Yes I agree it would be nice if this library's scope would be widened to support more formats.

**Comment by ekinhbayar** [comment]
I wonder about all the naming then. Including the repo/packagist.

**Comment by PeeHaa** [comment]
Yes this would probably warrant a rename.
